src/block-ciphers.rst

```diff
@@ -220,51 +220,51 @@ A closer look at Rijndael
 .. canned_admonition::
    :from_template: advanced
 
-AES consists of several independent steps. At a high level, AES is a
+AES includes several independent steps. At a high level, AES is a
 :term:`substitution-permutation network`.
 
 Key schedule
 ''''''''''''
 
-AES requires separate keys for each round in the next steps. The key
-schedule is the process which AES uses to derive 128-bit keys for each
+The next steps show how AES requires separate keys for each round. In the key
```

Comment:
This changes the meaning of the sentence. It's saying AWS consists of several steps (see before), and future steps require round keys, and this step is about generating those round keys. It does not show how AES "requires" round keys, it shows how AES produces them.